src/templates/_common/components/forms.jsx

```diff
@@ -135,10 +136,21 @@ export const FormRow = ({
             return 'gr-12';
         } else if (has_geovalidator) {
             return 'gr-3 gr-12-m';
+        } else if (label_row_cell_width) {
+            return `gr-${12 - label_row_cell_width} gr-12-m ${sub_row_class || ''}`;
         }
         return `gr-8 gr-12-m ${sub_row_class || ''}`;
     };
 
+    const getLebelRowClassName = () => {
```

Comment:
I think you did small typo here with the word `Lebel`, it should be `Label`

Reply:
> 

Thanks for your notes. I think both names are suitable, because I created a new attribute "label_row_cell_width" which is passed to parent component and used into this new function. I think this function is not only for setting width but also for generating common classes. How do you think ?

Reply:
> 

Sure! sounds good to me